### DISCUSSION TITLE
How to reuse a local micromamba environment when using Jupyter notebook?

### DISCUSSION THREAD

**Discussion by ShuhuaGao** [cjdoris/PythonCall.jl#466] [Q&A] [answered]
I have configured a micromamba environment _plot_ manually with all dependencies satisfied. And I want PythonCall (or CondaPkg) to use this environment. In terminal and Julia REPL, I can make it as follows:
```
PS C:\Users\lenovo> micromamba activate plot
(plot) PS C:\Users\lenovo> julia

julia> ENV["JULIA_CONDAPKG_BACKEND"] = "Null"
"Null"

julia> using PythonCall
```

Now I want to write code in a Jupyter notebook in VS code. How can I make the notebook live in the existing _plot_ Python environment?
Note that, in VS code, I simply create a file "hello.ipynb" and then open it.

**Comment by cjdoris** [comment, marked as answer]
I'm not sure - this seems more like a general VSCode notebooks question. Maybe try starting Code from within your Python environment?

**Reply by ShuhuaGao** [reply]
That's great! Just need to start VS code from the terminal by typing `code` after the proper python environment is activated.
